### PR TITLE
Feature/authentication

### PR DESCRIPTION
**Pull request by fangpinsern**

### CODE REVIEW

File: src/schema/authSchema.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const yup = require("yup");
+
+const loginSchema = yup.object().shape({
+  username: yup.string().required(),
+  password: yup.string().required(),
+});
+
+const signupSchema = yup.object().shape({
+  email: yup.string().required(),
```
Comment (chwennnn) [inline src/schema/authSchema.js:9]: this one can string().email() right

File: src/controllers/auth/register.controller.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+const { createUser } = require("../../services/users.service");
+
+const registerController = async (req, res, next) => {
+  const body = req.body;
+  console.log("INFO - Body is valid");
+  const email = body.email;
+  const username = body.username;
+  const password = body.password;
+
+  // Check if user exist
```
Comment (chwennnn) [inline src/controllers/auth/register.controller.js:16]: is this a to-do?

**Review by chwennnn — COMMENTED** [review]
the rest looks good

File: src/schema/authSchema.js (hunk 1 above)
Comment (fangpinsern) [reply src/schema/authSchema.js:9]: added

File: src/controllers/auth/register.controller.js (hunk 2 above)
Comment (fangpinsern) [reply src/controllers/auth/register.controller.js]: updated

**Review by chwennnn — APPROVED** [review]
looks good!

**Pull request merged**